Library/Homebrew/manpages/brew.1.md

```diff
@@ -298,6 +298,10 @@ Note that these flags should only appear after a command.
 
     If no <formulae> are given, check all installed brews.
 
+  * `migrate` [<formulae>]:
```

Comment:
`migrate` is currently like `postinstall` in that users shouldn't need to run it but it's there to help us with debugging user issues. If there are none we may kill it.